modules/charts/src/theme/nivo/config.tsx

```diff
@@ -0,0 +1,118 @@
+export const defaultNivoConfig = {
```

Comment:
Missing types for this. Should this be based on Nivo config type, or is this our own interface shape that we should define and export?